src/daos/metadata_dao.py

```diff
@@ -0,0 +1,115 @@
+import csv
+import os
+from datetime import datetime
+from typing import Dict, Any, List
+import logging
+
+logger = logging.getLogger(__name__)
+
+FIELD_NAMES = ['market_slug', 'market_id', 'asset_id', 'outcome_name', 'run_at', 'game_start_at', 'game_end_at']
+
+def _get_file_path(market_slug: str, test_mode: bool = False) -> str:
+    """Get the file path for the metadata CSV file."""
+    file_name = f"metadata_{market_slug}"
```

Comment:
Lets rename this to match the structure of the other files:

```suggestion
    csv_filename = os.path.join('data', f"{datetime.strftime('%Y%m%d')}_{market_slug}_market-metadata.csv")
```

datetime should be passed in like it is in the other daos.

Make sure to update anywhere else that is looking for a particular file structure to expect this format.